src/test/java/study1/racing/RacingTest.java

```diff
@@ -0,0 +1,46 @@
+package study1.racing;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import study1.racing.moveStrategy.TestMove;
+
+
+public class RacingTest {
+
+  @RepeatedTest(value = 100, name = "{displayName}, {currentRepetition}/{totalRepetitions}")
+  @DisplayName("ramdom 으로 값 생성 반복 확인 0 or 1")
+  public void checkMoveDistance(RepetitionInfo repetitionInfo) {
+    Car car = new Car();
+    int moved = car.move();
+    assertThat(moved).isGreaterThanOrEqualTo(0).isLessThanOrEqualTo(1);
+  }
```

Comment:
리팩토링 해보시면서 자동차 전진에 대한 테스트에 대해 다시 고민해보시면 좋을것 같아요

참고로 전략패턴을 활용하여 아래처럼 테스트 해볼 수 있겠네요 😄 
추가로 다음 미션 진행하시면서 Test 코드에서 `containsExactly()` 사용에 있어서 클래스에 `equals(), hashCode()` 메소드 재정의에 대해서 고민해보시면 좋을거 같아요.


```suggestion
    @DisplayName("Cars playCarGame 실행 시 car move 이동 테스트")
    @Test
    void playCarGame() {
        // WHEN
        cars.playCarGame(() -> true);

        // THAT
        assertThat(cars.getCars()).containsExactly(new Car(1), new Car(1), new Car(1));
    }
```